packages/grid/test/dynamic-item-size-polymer.test.js

```diff
@@ -0,0 +1,2 @@
+import '../all-imports.js';
```

Comment:
The `dynamic-item-size.test.js` only imports `vaadin-grid.js`, no need to use `all-imports` here.

Reply:
Done